doc/programming/language/modules.rst

```diff
@@ -66,3 +69,18 @@ implicitly public.
 
 Spicy comes with a set of :ref:`library modules <library>` that you
 may import in your code to gain access to their functionality.
+
+Global Properties
+-----------------
+
+A module may define the following global properties:
+
+    ``%byte-order = ORDER;``
+        Defaults the byte order for any parsing inside this module to
+        `<expr>`, where ``ORDER`` must be of typei is type
+        :ref:`spicy_ByteOrder`.
+
+    ``%spicy-version = "VERSION";``
```

Comment:
I think it would be better to call this `spicy-min-version` even now to leave the door open to some form of maximum version down the line to work around breaking API changes.

Reply:
Hmm, I don't see a max version going to be helpful (as we chatted about) because one won't be able to anticipate that number in advance. If we wanted to do support better control, we'd need some form of prerprocessors-tyle `if/else`. Given that, I prefer the shorter version here.